cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -791,7 +791,7 @@ def _get_extra_spec_disk_type(type_id):
         """
```

Comment:
The comment doesn't match the function anymore

Reply:
(one could argue to remove the whole line from the doc comment, since it's spelled out very clearly in the 3-line function body)